stremio-core-protobuf/proto/stremio/core/types/subtitle.proto

```diff
@@ -5,7 +5,8 @@ package stremio.core.types;
 option java_package = "com.stremio.core.types.subtitle";
 
 message Subtitle {
-  required string lang = 1;
-  required string url = 2;
-  optional string name = 3;
+  required string id = 1;
```

Comment:
Do we really need this `id` for subtitles? Don't see how it would be useful, since url is already a unique identifier for a resource? Having an unused field just increases deserialization burden, since there's usually quite a few subtitles for movies.

Reply:
The subtitles passed in a Stream response are passed to stremio-video (at least in stremio-web), this is why it's required as a field.
Otherwise subtitles pass from the stream resource itself are not loaded.

Reply:
Fyi: Core doesn't include these subtitles in the Loadable resource for the subtitles in the model

Reply:
But native apps won't use stremio video, so it's just a dead variable?

Reply:
I'm adding it for consistency and not to block I'll  leave it as is.

Reply:
PS: Not sure how the other platforms will implement the subtitles and fo rthe time being instead of discuss it I'm just adding it to move forward with the merge